### DISCUSSION TITLE
Question - Loss computation

### DISCUSSION THREAD

**Discussion by t-sanchez** [daviddmc/NeSVoR#4] [Q&A] [answered]
Hello Junshen,

I had a question on the computation of the losses. I saw in your code that you detach the `bias` and `c` computation for the computation of the variance. 
https://github.com/daviddmc/NeSVoR/blob/671c023305c04c47dbd7531fc3c92e85dbd1c0db/nesvor/nesvor/models.py#L310-L312

Is there a particular reason for it? Did you observe something like a more stable training? 

Thanks for your help!

Best,
Thomas

**Comment by daviddmc** [comment, marked as answer]
Hi Thomas,

Yes. Recently, we found that detaching the variables can stabilize the training in some cases. But for most of the data we evaluated, this doesn't change the performance much.
